CHANGELOG.md

```diff
@@ -17,6 +17,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   code
 - README has been updated with example SSLContext setup and it's use in Conjur
   class constructors
+- Maven pom.xml file includes configuration profiles for deploying to Sonatype OSSRH 
+(OSS Repository Hosting)  ([cyberark/conjur-api-java#6](https://github.com/cyberark/conjur-api-java/issues/6))
+- Introduced [upgrade instructions](https://github.com/cyberark/conjur-api-java/UPGRADING.md) to provide instructions for
+  upgrading to 3.0.0, or make use of published artifacts. These can be found in
+ `UPGRADING.md`. ([cyberark/conjur-api-java#77](https://github.com/cyberark/conjur-api-java/issues/77))
+
+### Changed 
+- Revised package references from `net.conjur.api` to `com.cyberark.conjur.api`
+- Revised visioning in `pom.xml` to use `SNAPSHOT` as the default
+- Revised directory structure for `src`, reflecting the new package namespace
```

Comment:
is this relevant to end users? if not it can be dropped

Reply:
Dropped